packages/ytfzf/ytfzf.patch

```diff
@@ -1,20 +1,38 @@
 --- a/ytfzf
 +++ b/ytfzf
-@@ -189,7 +189,7 @@
- : "${YTFZF_URL_HANDLERS_DIR:=$YTFZF_CONFIG_DIR/url-handlers}"
+@@ -168,7 +168,7 @@
+ 	# clean up only as parent process
+ 	#kill ytfzf sub process{{{
+ 	#i think this needs to be written to a file because of sub-shells
+-	jobs_file="${TMPDIR:-/tmp}/the-jobs-need-to-be-written-to-a-file.list"
++	jobs_file="${TMPDIR:-@TERMUX_PREFIX@}/the-jobs-need-to-be-written-to-a-file.list"
+ 	jobs -p > "$jobs_file"
+ 	while read -r line; do
+ 		[ "$line" ] && kill "$line" 2> /dev/null
+@@ -312,7 +312,7 @@
  : "${YTFZF_CUSTOM_THUMBNAILS_DIR:=$YTFZF_CONFIG_DIR/thumbnails}"
+ : "${YTFZF_EXTENSIONS_DIR:=$YTFZF_CONFIG_DIR/extensions}"
  
 -: "${YTFZF_SYSTEM_ADDON_DIR:=/usr/local/share/ytfzf/addons}"
-+: "${YTFZF_SYSTEM_ADDON_DIR:=@TERMUX_PREFIX@/share/ytfzf/addons}"
++: "${YTFZF_SYSTEM_ADDON_DIR:=@TERMUX_PREFIX@/local/share/ytfzf/addons}"
```

Comment:
`/usr/local/a` should be replaced with `@TERMUX_PRFIX/a`. I think `/local` is not necessary.